### ISSUE TITLE
Open Source Friday - Oqtane - 4-11-205

### ISSUE THREAD

**Issue by sbwalker** [githubevents/open-source-friday#117]
### Name

Shaun Walker

### GitHub Handle

@sbwalker

### Tell us about yourself

I am the original creator of DotNetNuke, one of the pioneering open source projects native to the Microsoft platform originally released in 2003. I am also the creator of Oqtane, an open source web application framework for modern .NET. I have been a Microsoft Most Valuable Professional (MVP) for 16 years and was recognized in 2011 as a leading entrepreneur in the Business In Vancouver (BIV) Forty Under 40 business awards. I am also the Chair of the Project Committee for the .NET Foundation and have been a volunteer for the .NET Foundation since 2014. 

### Project Name

Oqtane Framework

### Project Repo Link

https://github.com/oqtane/oqtane.framework

### Stream Date

- [ ] Yes
- [X] Not yet

### Twitter URL

@sbwalker

### LinkedIn URL

https://www.linkedin.com/in/shaunbrucewalker/

### Additional Information

I would like to talk about my latest open source project, Oqtane.... but I am also happy to talk about other open source topics (ie. https://youtu.be/VYi7GggFyxc?t=12919)

**Comment by sbwalker** [comment]
Is there a process for discussing dates for these sessions?

**Comment by LadyKerr** [comment]
Hey @sbwalker, we've been on vacation and just getting back!

Are you available in March?

**Comment by sbwalker** [comment]
@LadyKerr yes, I am available in March (except for Mar 24-28 which is the Microsoft MVP Summit in Redmond)

**Comment by LadyKerr** [comment]
Hey @sbwalker thanks for your patience!
Please choose a date from the cal: https://gh.io/osf-booking

**Comment by AndreaGriffiths11** [comment]
Hi @sbwalker does April 11 work?

**Comment by sbwalker** [comment]
@AndreaGriffiths11 yes April 11 works... I already accepted the calendar invite

**Comment by AndreaGriffiths11** [comment]
Perfect! Thank you very much for confirming

**Comment by AndreaGriffiths11** [comment]
Complete: https://studio.youtube.com/video/j7WID3q1yY0/edit
